lib/fluent/plugin/in_tail.rb

```diff
@@ -892,16 +967,35 @@ def opened?
         def handle_notify
           with_io do |io|
             begin
+              number_bytes_read = 0
+              start_reading = Fluent::EventTime.now
               read_more = false
 
               if !io.nil? && @lines.empty?
                 begin
                   while true
                     @fifo << io.readpartial(8192, @iobuf)
                     @fifo.read_lines(@lines)
-                    if @lines.size >= @read_lines_limit
+
+                    number_bytes_read += 8192
```

Comment:
`8192` could become a variable that is also used on line 977 

but not a big deal

Reply:
I think that they should be a constant instead of `8192` or variables.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3185/commits/8d26d11073baa140e93e0a39765e732a0e78f11f.